rest-api/tools/import_organizations.sh

```diff
@@ -25,6 +25,7 @@ then
   source tools/auth_setup.sh
   run_cloud_sql_proxy
   set_db_connection_string
+  get_geocode_key
```

Comment:
The else block here -- when ${PROJECT} is not set -- is when we're just dealing with the local database. 

It still seems like in that case you'll need to get and use the geocode key?